### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by javierbustillo**

### CODE REVIEW

File: CreateDB [hunk 1]
```diff
@@ -0,0 +1,53 @@
+CREATE TABLE Users (
+    uid SERIAL PRIMARY KEY,
+    username varchar(30) UNIQUE NOT NULL,
+    hashed_password varchar(100) NOT NULL,
+    created_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE Chat_Group (
```
Comment (ChristianPerez34) [inline CreateDB]: A chat group also has participants. Shouldn't we add a participants column?

File: app.py [hunk 2]
```diff
@@ -8,5 +11,81 @@ def hello_world():
     return 'Hello World!'
 
 
+@app.route('/register', methods=['POST'])
+def create_user():
+    """
+    Should create user
```
Comment (ChristianPerez34) [inline app.py]: Should be a declaration of what method does, not a posibility

File: app.py [hunk 3]
```diff
@@ -8,5 +11,81 @@ def hello_world():
...
+        password = request.form['password']
+        hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+        return jsonify(username=username, msg='Success'), 201
+    return jsonify(msg='Error'), 500
+
+
+@app.route('/login', methods=['POST'])
+def login():
+    """
+
```
Comment (ChristianPerez34) [inline app.py]: Missing documentation

File: app.py (hunk 2 above)
Comment (javierbustillo) [reply app.py]: It's not actually creating a user right now, it should create one phase 2

File: config/dbconfig.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+pg_config = {
+    'user' : 'postgres',
```
Comment (ChristianPerez34) [inline config/dbconfig.py]: User should not be postgres

**Review by ChristianPerez34 — CHANGES_REQUESTED** [review]
Resolve conflicts

**Pull request merged**